README.md

```diff
@@ -10,19 +10,18 @@ These instructions will get you a copy of the project up and running on your loc
 ### Prerequisites
 
 * [CENTOS 6/7](https://wiki.centos.org/) - Linux Distribution based on Red Hat
-* [PHP 7.1](http://php.net/docs.php) - General-purpose scripting language
-* [PostgreSQL 9.6](https://www.postgresql.org/docs/9.6/static/release-9-6.html) - ORDBMS
+* [PHP 7.1+](http://php.net/docs.php) - General-purpose scripting language
+* [PostgreSQL 9.6+](https://www.postgresql.org/docs/9.6/static/release-9-6.html) - ORDBMS
 * [Elasticsearch 6.7.2+](https://www.elastic.co/products/elasticsearch) - ElasticSearch Document Indexer
 * [RabbitMQ](https://www.rabbitmq.com/documentation.html) - Open source message broker software
-* [Composer 1.5.2+](https://getcomposer.org/) - Dependency Manager for PHP
-* [Yarn 1.15.2](https://yarnpkg.com/en/) - Package Manager
+* [Composer 1.9.0+](https://getcomposer.org/) - Dependency Manager for PHP
+* [Yarn 1.19.1+](https://yarnpkg.com/en/) - Package Manager
 
 ### Installation
 
-Pelagos is a [Symfony 3.4](https://symfony.com/doc/3.4/index.html) project, please follow the normal configuration regarding setting up your webserver for a Symfony project.
+Pelagos is a [Symfony 4.3+](https://symfony.com/doc/4.3/index.html) project, please follow the normal configuration regarding setting up your webserver for a Symfony project.
 
 To install fullfill prerequisites then run:
-* `export SYMFONY-ENV=drupal_prod`
 * `composer install`
 * `yarn install`
 * `yarn run encore production`
```

Comment:
I say we get rid of the name list, and just the link. Wadda ya think?

Reply:
Yes, we can get rid of that. We've not mentioned James and others who had contributed to the project.

Reply:
Name list in the readme? I think we either remove it or fully populate it.